vue-components/stories/TooglePoC.stories.ts

```diff
@@ -0,0 +1,57 @@
+import TogglePoC from '@/components/TogglePoC';
+import { Component } from 'vue';
+
+export default {
+	component: TogglePoC,
+	title: 'TogglePoC',
+};
+const options = [
+	{
+		label: 'Pineapple',
+		value: 'pineapple',
+	},
+	{
+		label: 'Mushrooms',
+		value: 'mushrooms',
+	},
+	{
+		label: 'Spinach',
+		value: 'spinach',
+	},
+	{
+		label: 'Artichoke',
+		value: 'artichoke',
+	},
+];
```

Comment:
I see where you're coming from regarding the options and not being convinced by the single value case, although I think it's a fair compromise to make for the sake of cohesion and simplicity. 

We can add a ticket to the DS board, if we want to, to enable support for a single object or even a single string in the future.

Reply:
In any case, we can already say that by default the value could be equal to the label if not explicitly set. At least that's the way I would expect it to work.

Reply:
Yep, that makes sense